rafx-assets/src/assets/asset_manager.rs

```diff
@@ -279,6 +279,25 @@ impl AssetManager {
             .map(|x| x.material_pass_resource.clone())
     }
 
+    pub fn get_material_pass_by_name(
```

Comment:
I've been thinking about having fewer functions (i.e. eventually removing accessors like get_material_pass_by_index) on AssetManager. Maybe functions like this could be on the MaterialAsset and we just have a single function returning the &MaterialAsset. So someone could do `asset_manager.committed_asset(material_handle)?.get_material_pass_by_phase(...)`

Reply:
That's fine with me. I'll move them.